packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -231,6 +231,11 @@ export class PaymentChannelClient {
     return false; // only beneficiary may receive payments
   }
 
+  shouldSendSpacerState(channelState: ChannelState): boolean {
+    const turnNum = Number(channelState.turnNum);
+    return turnNum === 3 ? true : false;
```

Comment:
Suggestion: instead of hard coding 3 that looks like a magic number, assign it to a constant.

Reply:
400a5ea